distribution/lib/Standard/Base/0.0.0-dev/src/Data/Pair.enso

```diff
@@ -82,7 +84,8 @@ type Pair
             -2 -> self.first
             _ -> if_missing
 
-    ## Checks whether a predicate holds for at least one element of self pair.
+    ## GROUP Logical
+       Checks whether a predicate holds for at least one element of self pair.
```

Comment:
Unrelated to this PR, but going thru the docs I see this `self` is an accidental artifact of the `this/self` keyword rename.

Let's fix it :)

```suggestion
       Checks whether a predicate holds for at least one element of this pair.
```